conf/application.conf

```diff
@@ -195,20 +181,12 @@ controllers {
   }
 }
 
-fileTransfer {
-  enabled = true
-  frequencySeconds = 420
-  lockMinutes = 7
-}
-
-envelope.collection.name = "s3envelopeids"
 dvr.collection.name = "dvrRecords"
 dvr.record.ttl.duration = 14 days
 
 adhoc.mongo {
```

Comment:
do we still need adhoc mongo?

Reply:
i guess if you want to run adhoc mongo queries in prod, we keep it, if not, we remove it. I believe we shouldn't have had that in in the first place... but not my circus, not my monkey. I'll happily remove it if you want it gone 👍